plugins/bazaar/src/components/HomePage/HomePage.tsx

```diff
@@ -0,0 +1,241 @@
+
+const useStyles = makeStyles({
+  header: {
+    display: 'inline-block',
+    width: '60%',
+    fontSize: '1.7rem',
+    margin: '1rem 0',
+  },
+  link: {
+    display: 'inline-block',
+    width: '40%',
+    textAlign: 'end',
+    fontSize: '1.2rem',
+  },
+  hr: {
+    marginBottom: '1.3rem',
+  },
+  headerDiv: {
+    padding: '0 1rem',
+  },
+  container: {
+    marginTop: '2rem',
+  },
+  subheader: {
+    fontWeight: 'bold',
+  },
+  description: {
+    textAlign: 'justify',
+  },
+});
+
+import { useApi } from '@backstage/core-plugin-api';
+import { catalogApiRef } from '@backstage/plugin-catalog-react';
+import { Entity } from '@backstage/catalog-model';
+import { usePromiseTracker, trackPromise } from 'react-promise-tracker';
+
+export const HomePage = () => {
+  const location = useLocation();
+  const classes = useStyles();
+  const catalogApi = useApi(catalogApiRef);
+  const initValue: Entity[] = [];
+  const [value, setValue] = useState(initValue);
+  const { promiseInProgress } = usePromiseTracker();
+  const [notBazaarEntities, setNotBazaarEntities] = useState(initValue);
+  const [openAdd, setOpenAdd] = useState(false);
+  const [openNoProjects, setOpenNoProjects] = useState(false);
+
+  const handleCloseNoProjects = () => {
+    setOpenNoProjects(false);
+  };
+
+  useEffect(() => {
+    async function fetchEntities() {
+      const components = await catalogApi.getEntities({
+        filter: { kind: 'Component' },
+      });
+
+      const bazaarEntities = components.items.filter(e => e?.metadata?.bazaar);
+      setNotBazaarEntities(
+        components.items.filter(
+          e =>
+            !e?.metadata?.bazaar &&
+            e?.metadata?.annotations?.['backstage.io/edit-url'],
+        ),
+      );
+      setValue(bazaarEntities);
+    }
+
+    trackPromise(fetchEntities());
+  }, [location, catalogApi]);
+
+  const tabContent = [
+    {
+      label: 'Home',
+      component: (
+        <Content noPadding>
+          <ContentHeader title="Latest projects">
+            <SupportButton />
+          </ContentHeader>
+          <ProjectPreview entities={value || []} />
+          <Content noPadding className={classes.container}>
+            <ContentHeader title="All projects">
+              <Button
+                variant="contained"
+                color="primary"
+                onClick={() => {
+                  if (notBazaarEntities.length !== 0) {
+                    setOpenAdd(true);
+                  } else {
+                    setOpenNoProjects(true);
+                  }
+                }}
+              >
+                Add project
+              </Button>
+              <AddProjectDialog
+                entities={notBazaarEntities}
+                handleClose={() => {
+                  setOpenAdd(false);
+                }}
+                openAdd={openAdd}
+              />
+            </ContentHeader>
+            <ProjectCatalog entities={value || []} />
```

Comment:
I find the dual display here a bit strange 🤔  It seems like the volume might be low enough to handle just a list of cards, maybe paginated for extreme cases -- with sorting options for last updated, most members, etc.